internal/inmemoryfilelinux.go

```diff
@@ -0,0 +1,35 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+//go:build linux
+
+package internal
+
+import (
+	"fmt"
+	"golang.org/x/sys/unix"
+)
+
+func CreateMemfd(name string, data []byte) (int, error) {
+	fd, err := unix.MemfdCreate(name, unix.MFD_CLOEXEC|unix.MFD_ALLOW_SEALING)
+	if err != nil {
+		return 0, fmt.Errorf("Failed to create memfd '%s': %v", name, err)
```

Comment:
In go conventionally, error messages are lower-cased.

Reply:
addressed in 7bb75db8922561f592e62fa1c7bd05bc5b442903